gammapy/modeling/models/temporal.py

```diff
@@ -334,6 +334,93 @@ def integral(self, t_min, t_max, **kwargs):
         return integral / self.time_sum(t_min, t_max)
 
 
+class GeneralizedGaussianTemporalModel(TemporalModel):
+    r"""A generalized Gaussian temporal profile
+
+    ..math::
+            F(t) = A * exp( - (\frac{|t - t_{maxim}|}{\sigma_r}) ^ v)   for  t < t_max
+            
+            F(t) = A * exp( - (\frac{|t - t_{maxim}|}{\sigma_d}) ^ v)   for  t > t_max            
+
+    Parameters
+    ----------
+    A : `~astropy.units.Quantity`
+        Maximum intensity
+    t_maxim: `~astropy.units.Quantity`
+        The time of the pulse's maximum intensity.
+    sigma_r : `~astropy.units.Quantity`
+        Rise time constant.
+    sigma_d : `~astropy.units.Quantity`
+        Decay time constant.
+    v : `~astropy.units.Quantity`
+        Pulse sharpness -> lower values implies a more peaked pulse
+    
+    """
+
+    tag = ["GeneralizedGaussianTemporalModel", "gengauss"]
+
+    _t_max_default = Time("2000-01-01")
+    t_maxim = Parameter("t_maxim", _t_max_default.mjd, unit = "day", frozen=False)
+    sigma_r = Parameter("sigma_r", "1d", frozen=False)
+    sigma_d = Parameter("sigma_d", "1d", frozen=False)
+    v = Parameter("v", 2, unit = "", frozen=False)
```

Comment:
Maybe find a longer, more descriptive parameter name? Even a greek letter might be better...